### PR TITLE
Timeout Set for waitForBlockFunction in common.go

### LINKED ISSUE #15: Add timeout to `WaitForBlockCompletion` function
Currently, we are waiting indefinitely for a transaction to be included in a block. We must have a timeout to stop this indefinite waiting.

Ideally,

`timeout = STATELENGTH * BLOCKTIME`

### PR DESCRIPTION
**Pull request by Yashk767**
Resolve #15 

Added a timeout of ( StateLength* BlockTime )in waitForBlockCompletion function in common.go

### CODE REVIEW

**Review by ashish10677 — APPROVED** [review]
LGTM

**Review by SkandaBhat — CHANGES_REQUESTED** [review]

File: utils/common_test.go [hunk 1]
```diff
@@ -1,60 +0,0 @@
-package utils
```
[the remark this answers is not in the document]
Comment (Yashk767) [reply utils/common_test.go:1]: As this file would contain tests for merkle tree functions and the these tests are already present in the go-merkletree package.

File: utils/array_test.go [hunk 2]
```diff
@@ -49,6 +49,14 @@ func TestContains(t *testing.T) {
 			},
 			want: true,
 		},
+		{
```
[the remark this answers is not in the document]
Comment (Yashk767) [reply utils/array_test.go:52]: It was added just to test the edge case when both array and value is nil.

**Review by ashish10677 — APPROVED** [review]

**Review by SkandaBhat — APPROVED** [review]
LGTM
